### PR TITLE
#19673 - Added styles for Bulk Information dynamic dialog

### PR DESCRIPTION
**Pull request by ivorpad**
Related to: https://github.com/dotcms/core/issues/19089

Storybook: https://dotcms.github.io/core-web/?path=/story/dotcms-misc-bulkdialog--basic

### CODE REVIEW

File: src/app/view/components/_common/dot-bulk-information/dot-bulk-information.component.scss [hunk 1]
```diff
@@ -1,7 +1,47 @@
...
+    > .bulk-information__fail-item + * {
+        margin-top: $spacing-3;
+    }
+}
+
+.bulk-information__fail-message {
+    background: hsl(7, 68%, 96%);
+    color: $red;
+    padding: 0.2rem $spacing-3; // 0.2rem hard coded as $spacing-1 is a bit too much
+    border-radius: 25px;
```
Comment (fmontes) [inline src/app/view/components/_common/dot-bulk-information/dot-bulk-information.component.scss:24]: 50%? o rems?

File: src/app/shared/models/dot-action-bulk-result/dot-action-bulk-result.model.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ export interface DotActionBulkResult {
     skippedCount: number;
     successCount: number;
     fails: DotBulkFailItem[];
+    action?: string;
```
Comment (fmontes) [inline src/app/shared/models/dot-action-bulk-result/dot-action-bulk-result.model.ts:11]: We need to update this to pass it where this is getting used: starting here: https://github.com/dotCMS/core-web/blob/master/src/app/portlets/dot-templates/dot-template-list/dot-template-list.component.ts#L486

**Review by fmontes — CHANGES_REQUESTED** [review]

File: src/app/view/components/_common/dot-bulk-information/dot-bulk-information.component.html [hunk 3]
```diff
@@ -1,6 +1,15 @@
-<p data-testId="successful">{{ 'Successful' | dm }}: {{ data.successCount }}</p>
-<p data-testId="fails">{{ 'Fails' | dm }}: {{ data.fails?.length }}</p>
-<div class="bulk-information__fail-item" *ngFor="let item of data.fails">
-    <h5>{{ item.description }}</h5>
-    <p>{{ item.errorMessage }}</p>
+<p data-testId="successful" class="bulk-information__success-message">
+    {{ data.successCount }} template{{ data.successCount > 1 ? 's' : '' }} have been successfully
```
Comment (alfredo-dotcms) [inline src/app/view/components/_common/dot-bulk-information/dot-bulk-information.component.html:2]: message key?

File: src/app/view/components/_common/dot-bulk-information/dot-bulk-information.component.scss (hunk 1 above)
Comment (ivorpad) [reply src/app/view/components/_common/dot-bulk-information/dot-bulk-information.component.scss:24]: 50% is rounded.

File: src/app/shared/models/dot-action-bulk-result/dot-action-bulk-result.model.ts (hunk 2 above)
Comment (ivorpad) [reply src/app/shared/models/dot-action-bulk-result/dot-action-bulk-result.model.ts:11]: @fmontes I think that's the only place where it was missing.

**Pull request merged**